### PR TITLE
Add comment for skip worktree together with assume unchanged

### PR DESCRIPTION
**Pull request by gerhardol**
## Proposed changes

https://github.com/gitextensions/gitextensions/pull/11011#discussion_r1346540551

Used in FormCommit

## Test methodology 

Manual

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Review by RussKie — APPROVED** [review]
Is it possible to add a regression test?

File: GitCommands/Git/GitModule.cs [hunk 1]
```diff
@@ -2845,53 +2845,54 @@ public IReadOnlyList<GitItemStatus> GetTreeFiles(ObjectId commitId, bool full)
...
+                List<GitItemStatus> result = new();
+
+                foreach (string line in lsString.LazySplit('\n', StringSplitOptions.RemoveEmptyEntries))
                 {
-                    continue;
+                    // Both AssumeUnchange and SkipWorktree will use 's',
+                    // already handled in GetAssumeUnchangedFilesFromString()
+                    char statusCharacter = line[0];
+                    const char SkippedStatus = 'S';
+                    if (statusCharacter is not SkippedStatus)
```
Comment (gerhardol) [inline GitCommands/Git/GitModule.cs:2882]: Will change to also include files that have both statuses (is it possible?) and assume unchanged is not shown

**Comment by gerhardol** [comment]
> Is it possible to add a regression test?

There are no previous tests, I do not think it is worth it. This would be a very exotic issue.

**Review by mstv — APPROVED** [review]
LGTM, have not run

**Pull request merged**